src/op_mode/generate_ovpn_client_file.py

```diff
@@ -89,10 +100,22 @@
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument("-i", "--interface", type=str, help='OpenVPN interface the client is connecting to', required=True)
-    parser.add_argument("-a", "--ca", type=str, help='OpenVPN CA cerificate', required=True)
-    parser.add_argument("-c", "--cert", type=str, help='OpenVPN client cerificate', required=True)
-    parser.add_argument("-k", "--key", type=str, help='OpenVPN client cerificate key', action="store")
+    parser.add_argument(
+        "-i",
+        "--interface",
+        type=str,
+        help='OpenVPN interface the client is connecting to',
+        required=True,
+    )
+    parser.add_argument(
```

Comment:
But we get them from PKI as completion helper, we do not use this script directly as separate file
it generates from the command
```
vyos@vyos# run generate openvpn client-config interface vtun20 ca openvpn-ca certificate openvpn-client

```
Where [openvpn-ca](https://github.com/vyos/vyos-1x/blob/0bd50e7ba9bee727346fd6c6c763f9a2b26ee5aa/op-mode-definitions/generate-openvpn-config-client.xml.in#L27) and [openvpn-client](https://github.com/vyos/vyos-1x/blob/0bd50e7ba9bee727346fd6c6c763f9a2b26ee5aa/op-mode-definitions/generate-openvpn-config-client.xml.in#L35) are certificate names from the PKI path

Reply:
My point is, the interface already defines which CA to use. There is no reason to ask the user for it separately.

Reply:
To be honest, the `tls ca` is multi-value
```
vyos@r4# show interfaces openvpn vtun20 tls ca-certificate 
 ca-certificate ca
+ca-certificate cb
+ca-certificate cc
+ca-certificate cd
[edit]
vyos@r4# 
```

Reply:
I wonder if clients should also allow all the same CAs in that case.

Reply:
It will be behavior change and should be processed by a separate PR
If I understand correctly, your goal is replacing CLI op-mode to provide `client-cert` only as an arg
I.e Before:
```
generate openvpn client-config interface vtun20 ca openvpn-ca certificate openvpn-client
```
After:
```
generate openvpn client-config interface vtun20 certificate openvpn-client
```
This way all CA's will be included from `pki ca` based on the `vtunX` configuration
Do we really need it? If yes, it is better to do it in a separate PR

I never used several CA's for a client. Even in theory is it possible to sign a client cert by several CA's?

Reply:
Does client mode support multiple CAs? If there are multiple CAs in an organiazation, I suppose the assumption is that the _server_ cert can also be signed with any of them, and thus the client may need to verify the server cert against multiple CAs.

Reply:
Let's fix bugs without new options
The original bug https://vyos.dev/T6486 and https://vyos.dev/T6379
And both should solved in this change.